go/api/base_client.go

```diff
@@ -1579,6 +1579,14 @@ func (client *baseClient) Persist(key string) (Result[bool], error) {
 	return handleBooleanResponse(result)
 }
 
+func (client *baseClient) ZCount(key string, rangeOptions *options.ZCountRange) (Result[int64], error) {
```

Comment:
```suggestion
func (client *baseClient) ZCount(key string, rangeOptions *options.ZCountRange) (int64, error) {
```
return value is not nullable

Reply:
Update tests accordingly

Reply:
Updated.